### DISCUSSION TITLE
Shutdown of the old LinkAce forum

### DISCUSSION THREAD

**Discussion by Kovah** [Kovah/LinkAce#386] [Announcements]
Dear old forum members,

I decided to shut down the forum. The primary reason is the cost of the server needed to run Discourse, the software powering the forum. As Discourse needs quite much resources, a tiny server is not enough to run it. As LinkAce has no long-term monthly sponsors as the moment, I had to pay for all bills on my own.

**What will be the replacement?**

Discussions about the project, feature requests and support were moved to this [Discussions forum](https://github.com/Kovah/LinkAce/discussions) which is directly coupled with where the development happens.

**What does that mean for me and my topics?**

I have already migrated some of the topics to the discussions forum, mostly feature requests and general discussions. Most of the support requests are either solved already, will be deprecated in the near future, or were migrated to the new forum.

Before the community forum is shut down, I archived all existing topics [in the Wayback Machine](https://web.archive.org/web/sitemap/https://community.linkace.org/). You can explore all topics in the archive.

If you have any questions, please post them as a reply to this topic.

**Comment by HamzaMoiyadi** [comment]
Is this also the reason why the [linkace](https://www.linkace.org/) site throws an SSL error? Did you shut down the server itself?
Asking for learning purposes (^_^")

**Reply by Kovah** [reply]
The LinkAce website is working for me. It's hosted on Netlify.

**Reply by HamzaMoiyadi** [reply]
<img width="1680" alt="Screenshot 2022-06-05 at 5 06 19 PM" src="https://user-images.githubusercontent.com/17302799/172048589-00da8dcb-c4c4-486d-b9d7-4524afe26ae9.png">

This is what I see when I try to navigate there

**Reply by HamzaMoiyadi** [reply]
My bad, seems like an ISP issue. 
Switching ISPs makes it work. Weird.
